coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/RelToPrestoConverter.java

```diff
@@ -85,15 +84,6 @@ public Result visit(Project e) {
     e.getVariablesSet();
     Result x = visitChild(0, e.getInput());
     parseCorrelTable(e, x);
-    // base class method will return x if isStar(). That causes problems if any of the views
-    // alias column names - returning x will make it pass through to the base table and the aliasing
-    // information is lost which causes problems.
-    // Removing isStar() check caused regression with lateral views. Removing isStar() will
-    // create SELECT x from UNNEST(..). Presto does not like SELECT around UNNEST. Hence, we return
-    // 'x' for unnest() clauses without adding select around it.
-    if (isUnnestAll(e)) {
-      return x;
-    }
```

Comment:
Could you clarify which part of this change makes this part unnecessary?

Reply:
This is part of `Preserve correlation when converting to Trino` commit and the change making this in my opinion obsolete is addition of
```
rightLateral = SqlStdOperatorTable.LATERAL.createCall(POS, rightLateral);
```

in `RelToPrestoConverter.visit(Correlate)`